lib/fluent/event.rb

```diff
@@ -21,6 +21,10 @@ class EventStream
     include Enumerable
     include MessagePackFactory::Mixin
 
+    def records
```

Comment:
I think it's best to:
* change `Chunk#size` to return # of events, and make `#length` as alias of `#size`
* add `Chunk#bytesize` to return bytes of chunk content

But this changes the meaning of `chunk.size`, and it will break compatibility of plugins.